python/Glacier2/callback/server/simple_wake_up_service.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# Copyright (c) ZeroC, Inc.
+
+import asyncio
+import time
+import Ice
+from EarlyRiser import ButtonPressed, WakeUpService, AlarmClockPrx
+from datetime import datetime
+from common.time import toDatetime
+
+async def callRing(alarmClock: AlarmClockPrx, wakeUpDatetime: datetime):
+    """
+    Asynchronously call the ring method on the alarm clock proxy until the user presses the stop button.
+    The first ring is delayed until the specified wakeUpDatetime. Then the method will keep ringing every 10
+    seconds until the user presses the stop button.
+
+    Parameters
+    ----------
+    alarmClock : AlarmClockPrx
+        The proxy to the client's alarm clock.
+    wakeUpDatetime : datetime
+        The datetime when the alarm should ring.
+    """
+    delay = wakeUpDatetime.timestamp() - time.time()
+    if delay > 0:
+        await asyncio.sleep(delay)
+
+    # First ring. This invocation reuses the connection established by the client.
+    buttonPressed = await alarmClock.ringAsync("It's time to wake up!")
+
+    # Keep ringing every 10 seconds until the user presses the stop button.
+    while buttonPressed is ButtonPressed.Snooze:
+        await asyncio.sleep(10)
+        buttonPressed = await alarmClock.ringAsync("No more snoozing!")
+
+
+class SimpleWakeUpService(WakeUpService):
+    """
+    SimpleWakeUpService is an Ice servant that implements Slice interface WakeUpService.
+    """
+
+    async def wakeMeUp(self, alarmClock: AlarmClockPrx, timestamp: int, current: Ice.Current) -> None:
+        """
```

Comment:
In C# and other mappings, we don't add a doc-comment to the servant methods because we rely on "inheritdoc".
We just add a regular comment that doesn't repeat everything. Can we do that in Python too?

Reply:
Not sure, I will review this in a follow-up PR.